core/src/main/java/com/redhat/lightblue/client/LightblueClient.java

```diff
@@ -12,6 +11,14 @@
 
     Locking getLocking(String domain);
 
+    /**
+     * This method allows
+     * @param domain
+     * @param usePost
+     * @return
+     */
+    Locking getLocking(String domain, Boolean usePost);
```

Comment:
I don't think there's a point to preserving the old lock API calls in the client. One ctor should be enough, using only POST.